MAINTAINING.md

```diff
@@ -66,26 +66,28 @@ less active when you don't. If you get a new job and get busy, that's alright.
 
 Releases should include documentation, git tag, bower package preparation and
 finally the actual npm module publish. We have all of this automated by running
-`./tools/release` from the root of the repository. __PLEASE DO NOT RUN `npm
-publish` BY ITSELF__. The release script will do that. We want to prevent issues
+`npm run release`. __PLEASE DO NOT RUN `npm
+publish` BY ITSELF__. The `release-script` will do that. We want to prevent issues
 like [#325](https://github.com/react-bootstrap/react-bootstrap/issues/325) and
 [#218](https://github.com/react-bootstrap/react-bootstrap/issues/218) from ever
-happening again. In order to run the release script you will need permission to
+happening again. In order to run the `release-script` you will need permission to
 publish the package to npm. Those with this permission are in the [publishers
 team](https://github.com/orgs/react-bootstrap/teams/publishers)
 
 *Note: The publishers team does exist. If you see 404 that means you just have no permissions to publish.*
 
-Example usages of the release script:
+Example usages of the `release-script`:
 
 ```bash
-$ ./tools/release patch
-$ ./tools/release minor
-$ ./tools/release major
-$ ./tools/release minor --preid beta   Use both bump and preid for first prerelease
-$ ./tools/release --preid beta         For follow on prereleases of the next version just use this
+$ npm run release patch
```

Comment:
According to official docs https://github.com/AlexKVal/release-script (look at the very end)
Nope, it shouldn't :smile: 
```
> npm run release patch
```
is right. I've checked it :wink: 

You need `--` double-dash only if you add `--<option-name>` options.